### ISSUE TITLE
Slots are not detected

### ISSUE THREAD

**Issue by m0jimo** [quasarframework/app-extension-qmediaplayer#51]
qmediaplayer v1.04

When slot is used
```html
<template v-slot:settings-menu>
          <div>
            <!--hide this button-->
           </div>
 </template>
```
it is ignored and settings-menu, volume and others components are not replaced with custom content.
![image](https://user-images.githubusercontent.com/37825539/62513336-ff65cc00-b81b-11e9-8a47-b1ab8ea4dafc.png)

**Comment by dev-steverob** [comment]
I think I have found the issue. I'm no expert, but was trying to replace the duration-time slot for live broadcasts I have logged what I've found here [https://forum.quasar-framework.org/topic/4102/how-to-use-slots-on-qmediaplayer](https://forum.quasar-framework.org/topic/4102/how-to-use-slots-on-qmediaplayer)

**Comment by hawkeye64** [comment]
Thanks for this. Will be fixed in v1.0.5

**Comment by hawkeye64** [comment]
Release in v1.0.5 - repoen if you are still having issues.
